web/src/modules/common/components/Page.js

```diff
@@ -53,7 +54,7 @@ const Page = ({
       {defaultThumbnailSrc && <Thumbnail alt='' src={defaultThumbnailSrc} srcSet={thumbnailSrcSet} />}
       <Caption title={title} />
       {children}
-      <RemoteContent dangerouslySetInnerHTML={{ __html: content }} onInternalLinkClick={onInternalLinkClick} />
+      <RemoteContent dangerouslySetInnerHTML={{ __html: sanitize(content) }} onInternalLinkClick={onInternalLinkClick} />
```

Comment:
Nice :+1:

Reply:
We already sanitize content :sweat_smile: 

https://github.com/Integreat/integreat-app/blob/main/api-client/src/endpoints/createCategoriesEndpoint.js#L30

I kind of favor dompurify though. We should look into which library we should use though.

Implementing it in the api-client has the advantage that the native app is also automatically sanitized. Maybe we should create a separate task though for sanitisation again. Do we sanitize the news content for example?

Reply:
oh :D i forgot to check all the deps in package.json.
But i def would recommend using a lib which is getting frequently updated, even there are lots of options regarding sanitizers.
Implementing in api client def makes sense maybe defining which fields can contain html content instead of regex testing